demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -75,13 +76,64 @@ def match(_dict: dict, path: Path) -> bool:
     def pack_id(self) -> str:
         return self.in_pack.pack_id if self.in_pack else ""
 
-    @property
-    def support_level(self) -> str:
-        return (
-            self.in_pack.support_level
-            if self.in_pack and self.in_pack.support_level
-            else ""
+    @validator("pack", always=True)
+    def validate_pack(cls, v: Any, values) -> Optional["Pack"]:
+        # Validate that we have the pack containing the content item.
+        # The pack is either provided directly or needs to be located.
+
+        pack = cls.get_pack(
+            values.get("pack"), values.get("relationships_data"), values.get("path")
         )
+        if v and not isinstance(v, fields.FieldInfo):
+            return v
+        return pack
+
+    @property
+    def in_pack(self) -> Optional["Pack"]:
+        """
+        This returns the Pack which the content item is in.
+
+        Returns:
+            Pack: Pack model.
+        """
+        if pack := ContentItem.get_pack(self.pack, self.relationships_data, self.path):
+            self.pack = pack
+        return pack  # type: ignore[return-value]
```

Comment:
you can do:
```suggestion
        if not self.pack:
            self.pack = ContentItem.get_pack(self.pack, self.relationships_data, self.path)
        return pack  # type: ignore[return-value]
```
so self.in_pack will not always run ContentItem.get_pack()